lighthouse-viewer/app/src/main.js

```diff
@@ -43,3 +43,10 @@ Promise.all(loadPolyfillPromises).then(_ => {
   // eslint-disable-next-line no-new
   new LighthouseViewerReport();
 });
+
+if ('serviceWorker' in navigator) {
+  navigator.serviceWorker.register('sw.js').catch(e => {
+    // eslint-disable-next-line no-console
```

Comment:
Just removed the `.catch` for now. We don't care if this shell sw fails to register.